applepay/utils.py

```diff
@@ -1,20 +1,38 @@
+"""
+The utils in this module are used to verify and parse a decoded
+apple pay token.  See the #ApplePaySpec below for more documentation
+and guidelines on the steps taken here.
+
+#ApplePaySpec: https://developer.apple.com/library/content/documentation/PassKit/Reference/PaymentTokenJSON/PaymentTokenJSON.html#//apple_ref/doc/uid/TP40014929-CH8-SW2
+"""
+
+
 import base64
-from datetime import timedelta
+import binascii
+from datetime import timedelta, datetime
+from pytz import utc
```

Comment:
pytz isn't really a builtin, so I think if we want to follow pep8 here, we need to move it down the the second section: https://www.python.org/dev/peps/pep-0008/#imports